cmd/bucket-handlers.go

```diff
@@ -715,7 +715,7 @@ func (api objectAPIHandlers) PostPolicyBucketHandler(w http.ResponseWriter, r *h
 		return
 	}
 	if objectAPI.IsEncryptionSupported() {
-		if hasServerSideEncryptionHeader(formValues) && !hasSuffix(object, SlashSeparator) { // handle SSE-C and SSE-S3 requests
+		if crypto.IsRequested(formValues) && !hasSuffix(object, SlashSeparator) { // handle SSE-C and SSE-S3 requests
```

Comment:
nit: comment should include SSE-KMS, right?

Reply:
In general yes - but since we do not support SSE-KMS we would have failed before it the user would have requested SSE-KMS. Anyway, changing the comment to `handle SSE requests`. :+1: